packages/airnode-deployer/src/utils/validation.ts

```diff
@@ -2,22 +2,31 @@ import { Config } from '@api3/airnode-node';
 import { validateJsonWithTemplate } from '@api3/airnode-validator';
 import * as logger from '../utils/logger';
 
-export function validateConfig(config: Config, nodeVersion: string, skipVersionCheck: boolean) {
-  if (!skipVersionCheck && nodeVersion !== config.nodeSettings.nodeVersion) {
-    logger.fail(
-      `nodeVersion under nodeSettings in config.json is ${config.nodeSettings.nodeVersion} while the deployer node version is ${nodeVersion}`
-    );
-    throw new Error("Node version specified in config.json does not match the deployer's version");
+export function validateConfig(config: Config, nodeVersion: string) {
+  if (config.nodeSettings.skipValidation) {
```

Comment:
Since `skipValidation` is optional I'm guessing that when it's not present in config.json then it means that we don't want to skip config validation.

Reply:
Yes, `skipValidation` is optional, so this works as well